flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/NettyPartitionRequestClient.java

```diff
@@ -128,11 +128,18 @@ public void operationComplete(ChannelFuture future) throws Exception {
                         if (!future.isSuccess()) {
                             clientHandler.removeInputChannel(inputChannel);
                             SocketAddress remoteAddr = future.channel().remoteAddress();
+                            String address =
+                                    remoteAddr == null
```

Comment:
I think perhaps we could direct output the connect id to simplify the implementation, since the address here should be equals to the one in the connection id.

Reply:
Thanks for the review, @gaoyunhaii. I've already improved it according to your suggestion.